src/next-app/pages/appointment-details.tsx

```diff
@@ -0,0 +1,273 @@
+                  fieldName="customer.phone_number"
+                  fieldLabel="Phone Number"
+                  fieldType="String"
+                  fieldDisabled={true}
+                  onChange={form.handleChange}
+                />
+                <CardTextField
+                  fieldValue={appointment.customer.email}
+                  fieldName="customer.email"
+                  fieldLabel="Email"
+                  fieldType="String"
+                  fieldDisabled={true}
+                  onChange={form.handleChange}
+                />
+                <h2 className="form-header">Vehicle Information</h2>
+                <CardTextField
+                  fieldValue={vehicle.manufacturer}
+                  fieldName="vehicle.manufacturer"
+                  fieldLabel="Manufacturer"
+                  fieldType="String"
+                  fieldDisabled={true}
+                  onChange={form.handleChange}
+                />
+                <CardTextField
+                  fieldValue={vehicle.model}
+                  fieldName="vehicle.model"
+                  fieldLabel="Model"
+                  fieldType="String"
+                  fieldDisabled={true}
+                  onChange={form.handleChange}
+                />
+                <CardTextField
+                  fieldValue={vehicle.year}
+                  fieldName="vehicle.year"
+                  fieldLabel="Year"
+                  fieldType="String"
+                  fieldDisabled={true}
+                  onChange={form.handleChange}
+                />
+                <CardTextField
+                  fieldValue={vehicle.color}
+                  fieldName="vehicle.color"
+                  fieldLabel="Color"
+                  fieldType="String"
+                  fieldDisabled={true}
+                  onChange={form.handleChange}
+                />
+                <CardTextField
+                  fieldValue={vehicle.vin}
+                  fieldName="vehicle.vin"
+                  fieldLabel="VIN"
+                  fieldType="String"
+                  fieldDisabled={true}
+                  onChange={form.handleChange}
+                />
+                {inEdit && <Button type="submit" title="Save" width={'100%'}></Button>}
+              </div>
+            </form>
+          </FormikProvider>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+export const getServerSideProps: GetServerSideProps<{}> = async (context) => {
+  const { id } = context.query;
+  //const id = 1;
+  const parsedCookies = cookie.parse(String(context.req.headers.cookie));
+  const access_token = parsedCookies.access;
+  try {
+    const appointment = await axios.get(`${apiUrl}/shops/appointments/${id}`, {
+      headers: { Authorization: `JWT ${access_token}` },
+    });
+
+    const vehicle = appointment.data.vehicle
+      ? await axios.get(`${apiUrl}/vehicles/vehicles/${appointment.data.vehicle}/`, {
```

Comment:
This still doesn't work. `appointment.data.vehicle` appears to be being used as a number but it is actually the following object: 
![image](https://user-images.githubusercontent.com/18541493/227805176-5ca22ef6-4886-48f3-a37f-1f29eb033e97.png)